section3/reflection.md

```diff
@@ -2,12 +2,47 @@
 
 1. In Section 3, we removed even more supports and pushed you to learn a bit more independently, read through technical documentation, and use your resources more. How did that feel?
 
+It was fun!!  I feel like the real deal.
+
 2. Think of a web or mobile application you use regularly. Where is an example of a place that the developers probably use Arrays? Explain.
 
+Google Classroom: When creating an assignment I can add materials, links, documents, rubrics, etc.  All of this is a "list" of data that is stored within the assignment created in the classwork section (which is also technically it's own list and of course the whole thing exists int he structure that is google classroom itself which has its own child lists(stream, classwork, grades, people))
+
 3. In your own words, explain what an Array method is.
+An array method is a way of interacting with Arrays.  You can use an array method to create a method, or add or subtract from it.
+
+Arrays are like a container that you store like/similar things inside (like voting ages, or a grocery list).
+Array methods are how you can interact with the array.  Need to add to the array variable for a shopping list?  You can use a methd like `push()` to add on.  need to remove something from the array, use the method `pop()`.  These are just two examples.  There are other methods as well.  Some methods will count how many items you have inside your array, some will order your array least to greatest or alphabetically.
+
 
 4. What is the purpose of a loop? When we would use a loop in conjunction with array vs. without?
+I learned that a good rule of thumb with coding is to keep your code DRY (don't repeat yourself).
+
+Loops are a great way to run a line of code over and over again without having to manually type the code yourself.
+
+A loop is a code that will take a set of rules and apply them for a certain condition.
+
+An example of a loop in the real world would be
+Need to drink 160 oz of water every day.  This is equal to about 5 HydroFlasks (32 oz).  So I will continue drinking Hydro flask until the quantity drank is greater than or equal to 5 full HydroFlasks.
+
 
 5. If you had to teach a 5 year old about loops, how would you explain it?
+An example of a loop in the real world would be
+Need to drink 160 oz of water every day.  This is equal to about 5 HydroFlasks (32 oz).  So I will continue drinking Hydro flask until the quantity drank is greater than or equal to 5 full HydroFlasks.
+
 
 6. What questions do you still have about Arrays and loops?
+can you put more than 3 statements in a for loop?
+
+More specifically can your condtion statement have and && or || written in this second position?  
+
+example:
```

Comment:
This would be a great question to try out in a text editor and see how the code responds.